.gitignore

```diff
@@ -1,6 +1,4 @@
 /target
-/page/page.js
-/main.js
```

Comment:
Honestly I'm not a big fan of including build artifacts in the source code repo... Do you have strong opinion about that? ;)

Reply:
Yea me either. I'm was not sure how else to do it. The simple truth is Node needs js files. On the plus side the end user no longer has to download the compiler ;)

I did a little more searching and just found this: https://github.com/anmonteiro/lumo. How about I submit a new PR with lumo?

Reply:
Interesting, you mean using lumo for compiling cljs->js or as a runtime dep?

Anyway, what if don't include these 2 files in repo for now, and keep building with `lein cljsbuild`? The idea of this PR is that end user should be able to `npm install` it, and for development / releasing new package it seems ok to require current deps (Java, leiningen). We could look into lumo separately.

What do you think?

Reply:
Yea I've experimented with it today sounds good in practice but I can't get it to compile yet :/ I think it's not ready for complex projects yet. Also compile time would increase by several minutes. Maybe for smaller projects with less dependencies It might work better. 

If I understand correctly you only want to update the js files on a release? OK but they can't be in the .gitignore file either or NPM won't copy them, see my other comment below.

Reply:
I think I misunderstood. If we don't include the js files how is NPM going to get them? We can make the user do the compiling, or any other ideas?

Reply:
Yeah, so I would like to gitignore these files but upload them when publishing a package. From what I understand we can put them in .gitignore and also put them in .npmignore but here prefixed with `!` to negate ignoring.

> You can negate a pattern by starting it with an exclamation point !.

Reply:
Great Idea, It turns out that by just creating a .npmignore file the .gitignore file is not parsed by NPM 😄.